homeassistant/components/notion/__init__.py

```diff
@@ -36,6 +37,20 @@
 CONFIG_SCHEMA = cv.deprecated(DOMAIN, invalidation_version="0.119")
 
 
+@callback
+def async_guard_against_missing_task(func) -> Callable:
```

Comment:
hum this isnt async?

Reply:
It's a common practice to prefix callbacks with "async_" so that, when used throughout, future readers can be assured that the function is async-friendly.